images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -11,38 +11,77 @@ ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     PYTHONIOENCODING="UTF-8" \
     LANG="C.UTF-8" \
     LANGUAGE="C.UTF-8" \
-    LC_ALL="C.UTF-8"
-
-ARG IMAGES_DIR="/opt/tmp/images"
-RUN oryx prep --skip-detection --platforms-and-versions nodejs=14 --debug \
-    && echo "$DEBIAN_FLAVOR" \
-    && . /tmp/build/__goVersions.sh \
-    && downloadedFileName="go${GO_VERSION}.linux-amd64.tar.gz" \
-    && ${IMAGES_DIR}/retry.sh "curl -SLsO https://golang.org/dl/$downloadedFileName" \
-    && mkdir -p /usr/local \
-    && gzip -d $downloadedFileName \
-    && tar -xf "go${GO_VERSION}.linux-amd64.tar" -C /usr/local \
-    && rm -rf $downloadedFileName
-
-RUN set -ex \
-    # Install Python SDKs
-    # Upgrade system python
-    && PYTHONIOENCODING="UTF-8" \
-    # It's not clear whether these are needed at runtime...
-    && apt-get update \
+    LC_ALL="C.UTF-8" \
+    ORYX_PATHS="/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin"
+    
+RUN apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
+        git \
+        make \
+        unzip \
+        # The tools in this package are used when installing packages for Python
         build-essential \
+        moreutils \
         python3-pip \
         swig \
         tk-dev \
+        unixodbc-dev \
         uuid-dev \
-    && rm -rf /var/lib/apt/lists/*
+        # Required for PostgreSQL
+        libpq-dev \
+        # Required for mysqlclient
+        default-libmysqlclient-dev \
+        # Required for ts
+        moreutils \
+        rsync \
+        zip \
+        tk-dev \
+        uuid-dev \
+        #TODO : Add these to fix php failures. Check if these can be removed.
+        libargon2-0 \
+        libonig-dev \
+        libedit-dev \
+    && rm -rf /var/lib/apt/lists/* \
+    # This is the folder containing 'links' to benv and build script generator
+    && mkdir -p /opt/oryx
+ARG IMAGES_DIR="/opt/tmp/images"
+ARG BUILD_DIR="/opt/tmp/build"
 
+
+RUN set -ex \
+ && yarnCacheFolder="/usr/local/share/yarn-cache" \
+ && mkdir -p $yarnCacheFolder \
+ && chmod 777 $yarnCacheFolder \
+ && . ${BUILD_DIR}/__nodeVersions.sh \
+ && if [ "${DEBIAN_FLAVOR}" == "bullseye" || "${DEBIAN_FLAVOR}" == "buster" ]; then ${IMAGES_DIR}/installPlatform.sh nodejs ${NODE16_VERSION}; fi \
```

Comment:
I imagine we don't have anything better, but just asking in case we're lucky: do we have any variable that would allow us to use the current default Node version, rather than hardcoding it to Node 16? That would be one less thing to remember to update in the future.

Reply:
Yep not sure if we have something as of today to keep track of the default version being used. Briefly chatted with @Cormac about this.
We may be able to add a constant in constants.yml but since the default versions do not change that often, this may be a far-fetched solution.

Reply:
@daniv-msft -- Sneha and I synced about this and it would be do-able if we migrated away from the `defaultVersion.txt` files that we upload to storage and used constants generated via `constants.yaml`, which would allow us to easily reference these values within the Dockerfiles (like we're doing on line 57 above with the `__nodeVersion.sh` file.

However, in my opinion, since we only need to update the default version used for a given platform every 1-2 years, the amount of work needed to migrate to this new model wouldn't necessarily take precedence over the other things we want to take care of in the short term. I think filing a work item for someone in the future to take a look at when they have time would be a nice-to-have.

Reply:
Thank you both!